.travis.yml

```diff
@@ -23,15 +23,15 @@ before_script:
   - helm lint helm/botkube
 
 script:
+  - docker login -u "$DOCKER_USERNAME" -p "$DOCKER_PASSWORD"
   - make test
   - make
-  - make container-image
+  - make buildx-container-image-build
```

Comment:
@xUnholy I don't think pushing images on PRs is a good idea

Reply:
There doesn't seem to be any way to truly test these images if they're not published prior, I'd like to be able to ideally test an image before it becomes an official release. Were there no images being pushed from other branches other than the main branch?

Reply:
Otherwise, we can just remove the `--push` arg from the `buildx` command.

Reply:
The current approach we follow is: Whenever a PR gets merged, we build image with `:latest` and from develop and push it into dockerhub. So latest image tag always points to the develop branch. When we make the release, we build image with release tag once we tag the release. 
For testing changes, the developers can build test image and push it to their personal dockerhub repo. It doesn't have to be infracloudio repo.

Reply:
Okay, so the `buildx-container-image-release` is fine, I'll just remove the `--push` from the buildx-container-image-build so it can validate it still builds appropriately.

Reply:
Sounds good. Also can we remove `container-image` target?

Reply:
Yep, I'll do that now - wasn't sure if you'd like to keep so that clarifies that!